registry/src/pages/user.js

```diff
@@ -29,7 +29,7 @@ const UserPage = () => {
   }
```

Comment:
<img width="1217" alt="Screenshot 2023-04-29 at 11 31 13 PM" src="https://user-images.githubusercontent.com/43968121/235317705-32552a37-8462-48b1-8c9c-1c141acab2fd.png">

This might need a little more UI modifications.

Reply:
Reducing the `font-size` of namespace description and date should work. And adding the `padding` between the list items.